.eslintrc.js

```diff
@@ -30,6 +30,7 @@ const realConfig = {
     'import/prefer-default-export': 'off',
     'import/no-default-export': 'error',
     'react/prop-types': 'off',
+    'consistent-return': 'off',
```

Comment:
😍 ty, this has been annoying me

Reply:
Anything else we should remove?

Reply:
Happy with just that, can't think of any others off the top of my head 🤔